src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -74,7 +74,7 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->beforeNormalization()
-                ->ifTrue(function ($v) { return !isset($v['assets']) && isset($v['templating']) && class_exists(Package::class); })
+                ->ifTrue(fn ($v) => !isset($v['assets']) && isset($v['templating']) && class_exists(Package::class))
```

Comment:
Most of this closures can be `static`. They were in your [previous PR](https://github.com/symfony/symfony/pull/51121/files#diff-0ef45febdaaf1b66bf33827186af3cafaee167ba59c902b237c8492855bab73eL71) on short closures.

Reply:
less visual debt FTW, making them static wouldn't provide anything